### ISSUE TITLE
Right side STL lacks sockets

### ISSUE THREAD

**Issue by heyfixit** [Schievel1/dactyl_manuform_r_track#3]
I didn't notice that until I printed, was it intentional? https://github.com/Schievel1/dactyl_manuform_r_track/blob/main/STLs/right.stl

I think I'll manually create those or print a "patch" piece that I can stitch onto the right side.

**Comment by Schievel1** [comment]
Do you mean sockets for the hot swaps?

**Comment by heyfixit** [comment]
Oops, I should have included a picture for clarity. These guys:

![image](https://user-images.githubusercontent.com/41658450/196004827-da0325ff-5c0f-4876-88b9-d1f22e6c7249.png)

Did you cut/melt those holes in manually?

**Comment by Schievel1** [comment]
Hm no I put put them into to model. Let see if I can find the right model

**Comment by Schievel1** [comment]
Yup, looks like I fucked this up. I am searching for those models feverishly now. 
Sorry about that.

**Comment by camillevuillaume** [comment]
Have you been able to find the STL? The scad model that you included seems to be missing the trackball and the holes (it seems that two files are missing, m1.stl and p2.stl when I open the file with OpenSCAD).

**Comment by Schievel1** [comment]
No, sadly not. But at least the STL has the trackball. It is just lacking the holes for the USB and TRRS jack.

**Comment by heyfixit** [comment]
If I were to print this again, I'd edit those models to include USB and TRRS holes on both sides.  The left side STL looks like this:

![image](https://user-images.githubusercontent.com/41658450/222210441-2a38a38c-a356-4255-87ca-928ae53ccc7e.png)

It wasn't a deal breaker for me, I just ended up melting appropriately-sized holes for both sides.
